src/TextInput/TextInput.tsx

```diff
@@ -203,6 +203,11 @@ export class TextInput extends React.Component<Props, State> {
       propTypes,
     ) as (keyof typeof propTypes)[]);
 
+    // (a11y) The 'for' attribute of the label tag and the 'id' attribute
+    // of the input tag should match so screen readers can determine which
+    // label goes with which form element
```

Comment:
I might add something like the following to this comment, to make the intention of the line of code below clearer.

> If an 'id' isn't specified, we use 'name' instead for this purpose.